dencam/gui.py

```diff
@@ -103,14 +103,8 @@ def _update(self):
 
         self.recorder.update_timestamp()
 
-        if self.state.value == 1:
-            self.show_frame('NetworkPage')
-        elif self.state.value == 2:
-            self.show_frame('RecordingPage')
-        elif self.state.value == 3:
-            self.show_frame('SolarPage')
-        elif self.state.value == 4:
-            self.show_frame('BlankPage')
+        if self.state.value >= 0 and self.state.value <= 4:
```

Comment:
I think given the issue I am talking about with my other comments here (and if you do add a frontside 5th state to STATE_LIST as I suggested somewhere) that this needs to be 
```
>= 1
```